css/evaluations.css

```diff
@@ -0,0 +1,50 @@
+body{
+	background-color: white;
```

Comment:
Good question :) I had another color, and I decided to have it back to white. I removed it.